proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/common/GrpcValidator.java

```diff
@@ -106,16 +100,17 @@ public void validateInvisibleTime(long invisibleTime, long minInvisibleTime) {
     }
 
     public void validateTag(String tag) {
-        if (StringUtils.isNotEmpty(tag)) {
-            if (StringUtils.isBlank(tag)) {
-                throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot be the char sequence of whitespace");
-            }
-            if (tag.contains("|")) {
-                throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain '|'");
-            }
-            if (containControlCharacter(tag)) {
-                throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain control character");
-            }
+        if (StringUtils.isEmpty(tag)) {
+            return;
+        }
+        if (StringUtils.isBlank(tag)) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot be the char sequence of whitespace");
+        }
+        if (tag.contains("|")) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain '|'");
+        }
+        if (containControlCharacter(tag)) {
+            throw new GrpcProxyException(Code.ILLEGAL_MESSAGE_TAG, "tag cannot contain control character");
```

Comment:
Is this modification related to this issue? If not, you'd better move it to another issue.

Reply:
Ok, thanks for your review, I modified it and I will add it to a new issue.